src/forecast_clarify/config.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+dirs = dict(
+    figures     = os.path.relpath('src/data/processedfigures/'),
```

Comment:
Currently, when I run one of the notebooks, it will not find the parameter files since it takes the working directory (which is where the notebook is saved) and searches relative to that. It works when I change the working directory of the notebook. Is this ok for the packaging or do we have to define the "base" of the package somehow?

Reply:
Correct me if I am wrong, but is the data supposed to be a part of the package or not? Is it more beneficial to create a `load_dataset` method that takes the path as an argument? This way we can save the data anywhere on the pc and still access it. If the data is supposed to be a part of the package I would use relative path to the folder since it would be stored alongside the package it self.

Reply:
Good question. Might not be the cleanest solution, but I think it might be easiest to make the dataset part of the package and then I create a new version when I update the dataset with the model parameters (~once a year). My problem with the current version was that it uses paths relative to my working directory and not relative to the package base. So I can only run those parts of the code that read in some of the data from the base directory of the package.